src/core/species_mod.F90

```diff
@@ -46,6 +48,10 @@ module species_mod
       integer :: drydep_index         !< drydep index in drydep array
       integer :: photolysis_index     !< photolysis index in photolysis array
       integer :: gocart_aero_index    !< gocart_aero index in gocart_aero array
+
+      ! Concentration
+      real(kind=fp), ALLOCATABLE :: conc(:)             !< species concentration [v/v] or kg/kg
```

Comment:
If it's not something per volume, I wouldn't call it concentration. kg/kg would indicate mass fraction or mass mixing ratio.

Reply:
I think we still need to figure this out exactly what units we will need. I suggest resolving this later when things become more clear